internal/2017-10-montreal/pipeline-knowledge-sharing.md

```diff
@@ -12,13 +12,13 @@ Sanitization procedure -- > Raw data is stored forever, but we sanitize it befor
 
 There are some sanitization checks that don't pass due to formatting reasons.
 
-Eventually the data ends up in postgress database. This is not the best database, but it's good enough. As the raw data storage increases, we will need to consider an alternative.
+Eventually the metadata ends up in postgress database. This is not the best database, but it's good enough. As the raw data storage increases, we will need to consider an alternative.
 
 How to run this:
 
 The code is basically stored in a docker image, so you can use whatever software you like. Airflow is a bunch of python libraries with their own version and you don't want to have conflicts and resolve them. So you have to build a docker image to run the software.
 
-How do you add the docker image to a chain (like described below)? You upload the docker image to docker hub, and then make the pipeline run this image through operations with elevated privileges (Airflow does this through a wraper script, run with sudo). At the end of these basic checks and setups, docker runs something and that's it. 
+How do you add the docker image to a chain (like described below)? You upload the docker image to docker hub, and then make the pipeline run this image through operations with elevated privileges (we do this through a wrapper script, run with sudo). At the end of these basic checks and setups, docker runs something and that's it. 
```

Comment:
My understanding is that @darkk wrote a custom wrapper for sudo.

Reply:
yup